Feminine-Divine/periodo
# Prescriptions/Advice based on Quiz response

#### Description
Issue FIXES #22


#### Describe the changes you've made

Added a result card at the end of the quiz that toggles recommendations based on the response of the user in the quiz.



#### Describe if there is any unusual behaviour of your code(Write `NA` if there isn't)

Not responsive for all screen sizes.

#### Additional context (OPTIONAL)

The user gets recommendations based on the selection of each answer in the quiz : 
![quiz_ans_pic](https://user-images.githubusercontent.com/73497800/115156759-7a7c1e00-a0a3-11eb-9768-7c97fb5fffc6.png)
![quiz_3](https://user-images.githubusercontent.com/73497800/115156881-21f95080-a0a4-11eb-9315-40df913288a0.png)

If the user has selected all options indicating that she has no current difficulties, a corresponding recommendation card is displayed : 
![quiz_ans_pic_2](https://user-images.githubusercontent.com/73497800/115156776-91227500-a0a3-11eb-9030-3eef437c76fa.png)


#### Test plan (OPTIONAL)

- Go to the Quiz page
- Select answers of your choice in the quiz.
- Click on the 'Tell Me More' button to see custom results displayed.

#### Checklist

- [ ] I have added my name in the contributors list at the end of README.md file.
- [x] Open Source Program names - GSSoC'21
